src/content/index.md

```diff
@@ -67,5 +67,8 @@ Then run `webpack` on the command-line to create `bundle.js`.
 
 __[Get Started](/guides/getting-started)__ quickly in our __Guides__ section, or dig into the __[Concepts](/concepts)__ section for more high-level information on the core notions behind webpack.
 
+## Browser Compatibility
+webpack supports all browsers that are [ES5-compliant](https://kangax.github.io/compat-table/es5/) (IE8 and below are not supported). webpack needs `Promise` for `import()` and `require.ensure()`. If you want to support older browsers, you will need to [load a polyfill](/guides/shimming/) before using these expressions.
```

Comment:
Can you add a line break after the title. Then, it would be good to merge to me.